packages/contracts/test/exchange/internal.ts

```diff
@@ -79,6 +80,9 @@ describe('Exchange core internal functions', () => {
         overflowErrorForSendTransaction = new Error(
             await getRevertReasonOrErrorMessageForSendTransactionAsync(RevertReason.Uint256Overflow),
         );
+        divisionByZeroErrorForCall = new Error(
+            await getRevertReasonOrErrorMessageForSendTransactionAsync(RevertReason.DivisionByZero),
```

Comment:
This should be `getRevertReasonOrErrorMessageForCall` instead, which doesn't exists.

Reply:
Should be fixed with #1000. That should also fix the unit test currently failing in the CI.